### PR TITLE
sample commit 2

### PR DESCRIPTION
**Pull request by Gegobite**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ function App() {
           GitHub Codespaces <span className="heart">♥️</span> React
         </p>
         <p className="small">
-          Edit <code>src/App.js</code> and save to reload.
+          Edit <code>src/App.js</code> and save to reload this.
         </p>
         <p>
           <a
```
Comment (Gegobite) [inline src/App.js:15]: test comment

**Review by Gegobite — COMMENTED** [review]
1

File: src/App.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ function App() {
           GitHub Codespaces <span className="heart">♥️</span> React
```
Comment (Gegobite) [inline src/App.js:9]: yep

File: src/App.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ function App() {
...
         </p>
         <p className="small">
-          Edit <code>src/App.js</code> and save to reload.
+          Edit <code>src/App.js</code> and save to reload this.
         </p>
```
Comment (gegobite2) [inline src/App.js:13]: fix it

Comment (Gegobite) [reply src/App.js:13]: done

**Review by gegobite2 — APPROVED** [review]

**Pull request closed without merging**